model/admin/db.go

```diff
@@ -26,7 +26,7 @@ var (
 	schedulerKey     = bsonutil.MustHaveTag(ServiceFlags{}, "SchedulerDisabled")
 )
 
-var settingsQuery = db.Query(bson.M{idKey: systemSettingsDocID})
+var settingsQuery = db.Query(bson.M{idKey: SystemSettingsDocID})
```

Comment:
I guess there's no real problem with this but I find the queries-in-variables approach to be weird.